### ISSUE TITLE
AVANCE 1

### ISSUE THREAD

**Issue by maryjomvz** [maryjomvz/ProyectoFinalBioinf2017-II-Maryjo-Contreras-Luna#1]
avance 1

trabajaré con datos obtenidos de una plataforma de illumona

[avance 1.md.zip](https://github.com/maryjomvz/ProyectoFinalBioinf2017-II-Maryjo-Contreras-Luna/files/923920/avance.1.md.zip)

**Comment by AliciaMstt** [comment]
Abriste un issue en vez de ponerlo como un archivo markdown en dentro de la pestaña de código. Además en el listado de proyectos pusiste el enlace directo, sin que dijera "avances 1". Te pido que lo cambies para contarte los avances 1 en totalidad.

Tu propuesta de trabajo se ve bien. Solo recuerda que deberás utilizar blast en su formato de línea de comando.

**Comment by AzaleaGuerra** [comment]
Suena bien. ¿Son datos propios?

**Comment by maryjomvz** [comment]
hola

ya hice mis modificaciones pero creo que sigo haciando algo mal
HELP!!

saludos

**Comment by AzaleaGuerra** [comment]
No sé cómo creaste el repositorio. Quizá es más fácil que hagas un nuevo repo y ponle que te haga un read me. A ese le puedes cambiar el nombre a "Avances 1". Otra opción es que dejes el read me y creas un nuevo archivo md que sea "Avaces 1". Veo que en las tareas has hecho lo mismo de crear issues.

**Comment by AliciaMstt** [comment]
Es que estás poniendo todo en Issues, el contenido de tu repositorio debe ir en "code". Ve a esa pestaña.

**Comment by maryjomvz** [comment]
hola de nuevo!!!

volví a modificar la subida mi archivo, lo puedes revisar por fa?
creo  que ya quedo

gracias!!

**Comment by AliciaMstt** [comment]
No, sigues sin tener un repositorio NUEVO solo y solo para tu proyecto, que NO sea una copia de todo el repo de la clase. Te recomiendo que si no lo has visto, veas este tutorial y lo sigas paso por paso. https://guides.github.com/activities/hello-world/ solo tienes que cambiar el nombre de hello-world por algo como Proyectofinal y dentro de ESE reposotirio crea un archivo llamado avances1.md y ahí el contenido de tus avances. Avísanos por aquí o por correo si ya lo lograste hacer, si no, puedes buscarnos para que te expliquemos con la compu enfrente.
